src/classes/minikube-utils.ts

```diff
@@ -0,0 +1,58 @@
+import YamlUtils from "./yaml-utils";
+import { CommandNames, KubeComponent } from "../interfaces/types";
+import OsUtils from "./os-utils";
+
+/**
+ * Provides utilities for initialising a project into Minikube
+ */
+export default class MinikubeUtils {
```

Comment:
Are these utils exclusively for Minikube or should this be named KubeUtils?

Reply:
Changed name to KubeUtils